crap-code.kt

```diff
@@ -0,0 +1,26 @@
+package badcode.example
+class PerformanceFile {
+    val largeList = List(1000000) { it }
+    // Inefficient way to calculate the sum of elements
+    var sum = 0
+    fun calculateSum() {
+        for (i in 0 until largeList.size) {
+            sum += largeList[i]
+        }
+        // typo in println() function
+        prentln("Sum: $sum")
```

Comment:
Fix the typo in `println` function to avoid runtime errors.```suggestion
println("Sum: $sum")
```